packages/frontend/src/components/create-campaign/form/steps/pool-step/picker/index.tsx

```diff
@@ -38,7 +37,7 @@ export function PoolPicker({ value, dex, onChange }: PoolPickerProps) {
 
     const chain = useChainId();
     const baseTokens = useBaseTokens(chain);
-    const { pools, loading } = usePools(chain, dex?.slug);
+    const { pools, loading } = usePools(chain, protocol as SupportedDex);
```

Comment:
Why do we need the cast here... with the new typings I thought this would be handled by TS